curp/tests/common/curp_group.rs

```diff
@@ -38,24 +49,45 @@ impl SnapshotAllocator for MemorySnapshotAllocator {
     }
 }
 
+/// Rocks snapshot allocator
+pub(crate) struct RocksSnapshotAllocator;
```

Comment:
Why should we re-define a `RocksSanpshotAllocator` here rather than reusing the `RocksSnapshotAllocator` defined in `snapshot_allocator.rs`?

Reply:
Here is `curp`'s Integration test, `RocksSnapshotAllocator` is defined in `xline` crate, `curp` should not depend on `xline`

Reply:
snapshot_allocator.rs is defined in xline, curps's tests should not depends on xline

Reply:
But these two pieces of code are totally the same. Can we extract them into a shared part?

Reply:
Fixed